lib/ssh-ks/client_app.go

```diff
@@ -0,0 +1,345 @@
+package ssh_ks
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/config"
+	"io/ioutil"
+	"os"
+)
+
+// ClientApp represents one client and holds all necessary structures
+type ClientApp struct {
+	// This points to the client holding this structure
+	This *Client
+	// Config holds all servers and clients
+	Config *Config
+	// Private is our private key
+	Private abstract.Secret
+}
```

Comment:
A lot of "App" appears here while still being in a "lib" folder ... It seems a bit too much on lib and not enough on app/ssh_* (only code for parsing command line). Is it not possible to harmonize it a bit? Keep `Client` in lib and put `ClientApp` in app ? I don't know maybe it's much refactoring I can't tell.